Volume/Views/DebriefArticleView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  DebriefArticleView.swift
+//  Volume
+//
+//  Created by Amy Chin Siu Huang on 12/2/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import AppDevAnalytics
+import Combine
+import SwiftUI
+import SDWebImageSwiftUI
+
+struct DebriefArticleView: View {
+    @State private var cancellableShoutoutMutation: AnyCancellable?
+    @EnvironmentObject private var userData: UserData
+    let header: String
+    let article: Article
+
+    var body: some View {
+
+        VStack(spacing: 30, content: {
+            Header(header, .center)
+                .padding(.top, 24)
+
+            Group {
```

Comment:
What's the point of putting this in a group? You use groups for example to apply the same style to everything in the group, but you don't do anything with the group here.